include/World/Player.h

```diff
@@ -19,7 +23,15 @@ namespace CC::Entities
 
     class Player : public CGE::Entities::Entity
     {
+		//there are clearly better ways to do this
+		std::default_random_engine random_data;
+		bool hasShot;
         CGE::View::Camera &camera_;
+        Progression progression;
+        Inventory inventory;
+		void init();
```

Comment:
I'm starting to question if it's github that is showing those weird indentation.